src/Orleans.Core/Configuration/Options/ClusterMembershipOptions.cs

```diff
@@ -117,5 +117,10 @@ public class ClusterMembershipOptions
         /// Whether to extend the effective <see cref="ProbeTimeout"/> value based upon current local health degradation.
         /// </summary>
         public bool ExtendProbeTimeoutDuringDegradation { get; set; } = false;
+
+        /// <summary>
+        /// Whether to enable probing silos indirectly, via other silos.
+        /// </summary>
+        public bool EnableIndirectProbes { get; set; } = false;
```

Comment:
We will need to document this so people don't enable it on rolling upgrade when upgrading from < 3.4

Reply:
In the release notes, right?